lib/Dialect/Seq/Transforms/PassDetails.h

```diff
@@ -17,12 +17,15 @@
 #include "circt/Dialect/HW/HWOps.h"
 #include "circt/Dialect/SV/SVDialect.h"
 #include "circt/Dialect/Seq/SeqOps.h"
+#include "circt/Dialect/Seq/SeqPasses.h"
 #include "mlir/Pass/Pass.h"
 
 namespace circt {
 namespace seq {
 
-#define GEN_PASS_CLASSES
+#define GEN_PASS_DEF_LOWERSEQFIRRTLTOSV
```

Comment:
Is there a generic "lower all pass_def" so we don't have to define each pass' define?

Reply:
Currently there is no such macro but it would be definitely useful so i'll open MLIR issue.